File: src/main/kotlin/github/jk1/smtpidea/server/imap/ImapServerManager.java
package github.jk1.smtpidea.server.imap;

import github.jk1.smtpidea.server.ServerManager;
import github.jk1.smtpidea.server.smtp.ServerConfiguration;

/**
 * @author Evgeny Naumenko
 */
public class ImapServerManager implements ServerManager {

    @Override
    public void setConfiguration(ServerConfiguration configuration) {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    @Override
    public void startServer() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    @Override
    public void stopServer() {
        throw new UnsupportedOperationException("Not implemented yet");
    }

    @Override
    public boolean isRunning() {
        throw new UnsupportedOperationException("Not implemented yet");
    }
}

File: src/main/kotlin/github/jk1/smtpidea/server/imap/ImapServerManager.kt
package github.jk1.smtpidea.server.imap

import github.jk1.smtpidea.server.ServerManager
import github.jk1.smtpidea.config.Pop3Config

/**
 *
 */
public class ImapServerManager : ServerManager<Pop3Config>{

    override var running: Boolean = false

    override fun start() {
        throw UnsupportedOperationException("Not implemented")
    }

    override fun stop() {
        throw UnsupportedOperationException("Not implemented")
    }
}